src/common/LatencyTuner.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 The Android Open Source Project
+ * Copyright 2018 The Android Open Source Project
```

Comment:
Note that you don't have to update the year when editing existing files. New files need to receive the current year when added.

Reply:
Reverted, thanks for the info.